client/ayon_core/tools/sceneinventory/model.py

```diff
@@ -490,27 +505,26 @@ def set_hierarchy_view(self, state):
     def _is_outdated(self, row, parent):
         """Return whether row is outdated.
 
-        A row is considered outdated if it has "version" and "highest_version"
-        data and in the internal data structure, and they are not of an
-        equal value.
+        A row is considered outdated if it has no "version" or the "isOutdated"
+        value is True.
 
         """
         def outdated(node):
             version = node.get("version", None)
-            highest = node.get("highest_version", None)
 
             # Always allow indices that have no version data at all
-            if version is None and highest is None:
+            if version is None:
                 return True
 
             # If either a version or highest is present but not the other
             # consider the item invalid.
             if not self._hierarchy_view:
                 # Skip this check if in hierarchy view, or the child item
                 # node will be hidden even it's actually outdated.
-                if version is None or highest is None:
+                if version is None:
```

Comment:
This will now technically never occur because `version is None` is already checked and returned for above.

Reply:
Then remove it...

Reply:
I guess this can be just
```
def outdated(node):
    return node.get("isOutdated", True)
```

Reply:
Done with https://github.com/ynput/ayon-core/pull/202/commits/910c79f886274d66e616229ad88afd90d3f5aa90 and https://github.com/ynput/ayon-core/pull/202/commits/e735c5dc723bc127eba840a03569e7e0c3db05e9